src/main/java/seedu/address/ui/TaskCard.java

```diff
@@ -38,19 +38,26 @@ public class TaskCard extends UiPart<Region> {
     private Label category;
     @FXML
     private Label deadline;
+    @FXML
+    private Label email;
+    @FXML
+    private Label status;
 
     /**
      * Creates a {@code PersonCode} with the given {@code Person} and index to display.
      */
     public TaskCard(Task task, int displayedIndex) {
         super(FXML);
         this.task = task;
-        id.setText(displayedIndex + ". ");
-        taskName.setText(task.getName().toString());
-        description.setText(task.getDescription().toString());
+        id.setText(displayedIndex + ".");
+        taskName.setText("Name: " + task.getName().toString());
+        description.setText("Description: " + task.getDescription().toString());
         priority.setText("Priority: " + task.getPriority().toString());
-        category.setText(task.getCategory().toString());
-        deadline.setText(task.getDeadline().toString());
+        category.setText("Category: " + task.getCategory().toString());
+        deadline.setText("Deadline: " + task.getDeadline().toString());
+        email.setText("Email: " + task.getPersonEmailAddress().toString());
+        status.setText("Status: " + String.valueOf(task.getStatus()));
```

Comment:
Can we make this say "status: completed/ not completed" or "completed: true/false"? I think that would be more helpful than "status:false"

Reply:
Done